addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -14,6 +24,19 @@
                     {'invisible': [('code', '=', 'custom')]}
                 </attribute>
             </group>
+            <field name="pending_msg" position="after">
+                <div class="o_row replace_pending_msg" colspan="2"
+                     attrs="{'invisible': ['|', ('code', '!=', 'custom'), ('custom_mode', '!=', 'wire_transfer')]}">
+                    <button class="fa fa-refresh fa-fw btn-link"
+                            data-tooltip-touch-tap-to-show="true"
+                            data-tooltip-template="web.FieldTooltip"
+                            data-tooltip-info="{&quot;field&quot;:{&quot;help&quot;:&quot;Reset pending message&quot;}}"
+                            title="Reset Pending Message"
+                            name="action_recompute_pending_msg"
+                            string=" Reload Pending Message"
```

Comment:
```suggestion
                            string="Reload Pending Message"
```

Reply:
it's for visual purposes, otherwise the "Reload" is glued to the reload image

Reply:
Hm, ok, that's the easy fix, I'm not sure the translator will keep it though. If you find a class to add instead, it might be better, but if not, that's fine, I guess.

Reply:
good idea! I will search in that direction :)